package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dataprotocol-client",
-  "version": "1.2.3",
+  "version": "1.2.4",
   "description": "Client implementation for sqlops studio",
```

Comment:
Can you update the description while you're working on this? `Client data protocol implementation for Azure Data Studio`